lib/util/constants.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export const partitionKey = "x-opt-partition-key";
+export const sequenceNumber = "x-opt-sequence-number";
+export const enqueuedTime = "x-opt-enqueued-time";
+export const offset = "x-opt-offset";
+export const enqueuedTimeAnnotation = `amqp.annotation.${enqueuedTime}`;
+export const offsetAnnotation = `amqp.annotation.${offset}`;
+export const sequenceNumberAnnotation = `amqp.annotation.${sequenceNumber}`;
+export const message = "message";
+export const error = "error";
+export const statusCode = "status-code";
+export const statusDescription = "status-description";
+export const errorCondition = "error-condition";
+export const management = "$management";
+export const partition = "partition";
+export const partitionId = "partitionId";
+export const readOperation = "READ";
+export const TLS = "tls";
+export const defaultConsumerGroup = "$default";
+export const eventHub = "eventhub";
+export const cbsEndpoint = "$cbs";
+export const cbsReplyTo = "cbs";
+export const operationPutToken = "put-token";
+export const aadEventHubsAudience = "https://eventhubs.azure.net/";
```

Comment:
>"https://eventhubs.azure.net/"; [](start = 36, length = 31)

also valid on private/gov clouds?

Reply:
Nope. This is only valid for the Production cloud. Will be adding some convenience where the user would not have to specify any of this. The default environment will be AzureCloud. If they want to target any sovereign clouds then they should simply provide the name of the cloud and we will get the token audience.